scrapi/harvesters/vivo.py

```diff
@@ -0,0 +1,238 @@
+"""
+A VIVO harvester for the SHARE project
+
+This harvester makes a SPARQL query to the VIVO SPARQL endpoint,
+the information to acces to a VIVO endpoint must be provided in the local.py file.
+"""
+
+from __future__ import unicode_literals
+
+import json
+import logging
+
+from datetime import date, timedelta, datetime
+
+from six.moves import xrange
+from SPARQLWrapper import SPARQLWrapper, JSON
+
+from scrapi import settings
+from scrapi.settings.sparql_mapping import *
```

Comment:
I prefer not to have `import *`, could you at least namespace the commands (`from scrapi.settings import sparql_mapping as mapping` or something like that)?